src/stories/Form.stories.js

```diff
@@ -136,9 +151,11 @@ Primary.args = {
                 textPosition: "left"
             },
         ]
-        
+
     }
   ],
+  sectionLabel: "balbla",
+  formSection: "lol",
```

Comment:
Do these have any usage?

Reply:
no was for testing forgot to remove them